.circleci/config.yml

```diff
@@ -32,6 +33,32 @@ only_development: &only_development
         - release
         - master
 
+jobs:
+  push-schema-changes:
+    executor: node/build
+    steps:
```

Comment:
At first I was surprised to see a lot of python, hokusai, etc. setup for this job, if it's opening a PR for Eigen if there are schema changes. Then I realized this was first pulling in the schema from staging (using hokusai ENV vars).

Wondering about alternatives to that. The location this Circle job runs in _might_ be a directory with a checked out version of the repo, in which case you could just grab the `_schemaV2` file directly.

Another option might be, since this job runs after the `push-staging-image`, is to use a node fetch to get the file directly from: `https://github.com/artsy/metaphysics/blob/staging/_schemaV2.graphql`.

That might be preferable to the somewhat more roundabout (and setup intensive) way of getting ENV vars from the deploy environment and regenerating.

Reply:
i know it's a bit of a faff (took a good long while for me to figure out too)

but I want to be sure that schema we push to eigen really was built with the latest staging env vars, and we don't necessarily get that by just reading `_schemaV2.graphql` from the repo, since we can't guarantee everyone is running git hooks correctly all the time.

If we could add an upstream CI job that tests the schema (e.g. by running `yarn dump:staging` and then `git diff`), that might be a better way to do it.

Or maybe you can think of another way?